apps/web/src/views/CakeStaking/components/LockWeeksForm.tsx

```diff
@@ -71,15 +71,19 @@ const WeekInput: React.FC<{
     [onInput],
   )
 
-  const appendComponent = (
-    <Box width={40} mr={12}>
-      <Image src="/images/cake-staking/lock.png" height={37} width={34} />
-    </Box>
+  const appendComponent = useMemo(
```

Comment:
I think it would be unnecessary to add `useMemo` to jsx? If we really need to do the perf optimization I'd recommend to add `React.memo` to the underlying component